src/xrc/xmlres.cpp

```diff
@@ -1701,9 +1701,9 @@ static wxColour GetSystemColour(const wxString& name)
         SYSCLR(wxSYS_COLOUR_BTNTEXT)
         SYSCLR(wxSYS_COLOUR_INACTIVECAPTIONTEXT)
         SYSCLR(wxSYS_COLOUR_BTNHIGHLIGHT)
-        SYSCLR(wxSYS_COLOUR_BTNHILIGHT)
+        SYSCLR(wxSYS_COLOUR_BTNHIGHLIGHT)
```

Comment:
These names are part of the library interface too and changing them would break the existing XRC files using them, we should continue to accept them (because there is really no good reason not to).